test-app/presenter/src/commonMain/kotlin/com/example/redwood/testing/presenter/TestApp.kt

```diff
@@ -30,62 +31,60 @@ import app.cash.redwood.ui.dp
 import com.example.redwood.testing.compose.Button
 import com.example.redwood.testing.compose.Text
 
+private val screens = buildMap<String, @Composable TestContext.() -> Unit> {
+  put("Repo Search") { RepoSearch(httpClient) }
+  put("UI Configuration") { UiConfigurationValues() }
+}
+
+@Stable
+class TestContext(
+  val httpClient: HttpClient,
+)
```

Comment:
This will become a context receiver eliminating the need to explicitly pass any values in the lambdas above.